extensions/appcleaner/src/lib/app-utils.ts

```diff
@@ -1,22 +1,48 @@
 import type { Application } from "@raycast/api";
-import type { AppItem } from "./types";
 
 import fs from "fs";
 import path from "path";
 
-export function filterApps(apps: AppItem[]): AppItem[] {
-  return apps.filter((app) => app.isSystemApp === false).sort((a, b) => a.name.localeCompare(b.name));
+export function filterApps(apps: Application[]): Application[] {
+  return apps.filter((app) => isSystem(app.path) === false).sort((a, b) => a.name.localeCompare(b.name));
 }
 
 export function getIcon(app: Application): string {
```

Comment:
Can't we use [fileIcon](https://developers.raycast.com/api-reference/user-interface/icons-and-images#fileicon) instead?

Reply:
that's fixed now. thanks!